src/python/bin/aqueduct

```diff
@@ -206,6 +207,9 @@ def apiKey():
 def clear():
     execute_command(["rm", "-rf", base_directory])
 
+def version():
```

Comment:
How are we going to ensure that the package version logged here is in sync with the executables in `~/.aqueduct`?

Reply:
Yeah, in a subsequent PR, my plan is to persist this version information into `~/.aqueduct`, and then whenever users do `aqueduct server` or `aqueduct ui`, we will compare the version number here with the persisted version number, and if there is any discrepancies (`aqueduct version` is ahead of the one persisted in `~/.aqueduct`), we will then pull the right version from S3 to upgrade the binaries. Does this design make sense?